app/controllers/EmailController.scala

```diff
@@ -34,6 +34,14 @@ class EmailController @Inject()(emailService: EmailService)(implicit val ec: Exe
     case _ => Future.successful(BadRequest)
   }}
 
+  def insertWithResponse(): Action[AnyContent] = Action.async { implicit request => request.body match {
```

Comment:
The `insert()` function above was created for this purpose. If we need it to return an `Accepted` response then I'd recommend just amending that rather than adding a new function.